src/vl-alert.js

```diff
@@ -47,7 +47,9 @@ export class VlAlert extends vlElement(HTMLElement) {
 
   connectedCallback() {
     this.__processActionsElementVisibility();
+    this.__processTitleElementVisibility();
     this._actionsSlotElement.addEventListener('slotchange', () => this.__processButtons());
+    this._titleSlotElement.addEventListener('slotchange', () => this.__processTitleElementVisibility());
```

Comment:
Hiervoor kan nog een unit test toegevoegd worden, want wordt momenteel niet in de huidige testen gedekt?